### PR TITLE
Added feature to enable rtcp_mux for calls to SIP endpoints

### PR DESCRIPTION
**Pull request by sergey-safarov**

### CODE REVIEW

**Comment by sergey-safarov** [comment]
Related to https://github.com/2600hz/kazoo-configs-freeswitch/pull/3

**Comment by lazedo** [comment]
Hi, I think we should stay with freeswitch defaults. What's the reason for inverting the logic here?

**Comment by sergey-safarov** [comment]
Kazoo designed to define all CCV as default false.
Now rtcp_mux implemented like other vars.

About default FreeSwitch value.
We can define `true` value via kapps_config call as default, then nothing changed from FreeSwitch view.

**Comment by sergey-safarov** [comment]
@lazedo also please look Fs profile config. RTCP is disabled at sip-profile setting. And rtcp_mux value is not important.

**Comment by lazedo** [comment]
But why would we want to change the way freeswitch works by default?

**Comment by sergey-safarov** [comment]
@lazedo is will be acceptable if I set default rtcp_mux to `true` at kazoo level.
Then FreeSwitch behavior will same as current.

**Comment by lazedo** [comment]
Can you answer the question? Why would we want to change default behavior?

**Comment by sergey-safarov** [comment]
Because kazoo pass 'media -> rtcp_mux' of endpoint json is not possible to pass in current implementation.
You not ready to merge PR with workarround (bad code) in https://github.com/2600hz/kazoo/pull/3872.

**Comment by lazedo** [comment]
So the change would be to enable passing false from kazoo_endpoint. It seems to me that changing the default behavior because of this is not acceptable

**Comment by lazedo** [comment]
Maybe someone from the team can help you with #3872. I can try to help you but only on next week

File: core/kazoo_endpoint/src/kz_endpoint.erl [hunk 1]
```diff
@@ -1552,6 +1554,11 @@ maybe_set_call_forward({Endpoint, Call, CallFwd, CCVs}) ->
                        )
     }.
 
+-spec maybe_set_rtcp_mux(ccv_acc()) -> ccv_acc().
+maybe_set_rtcp_mux({Endpoint, Call, CallFwd, CCVs}) ->
+    RTCP_MUX = kz_json:get_binary_value([<<"media">>, <<"rtcp_mux">>], Endpoint, ?DEFAULT_RTCP_MUX),
```
Comment (fenollp) [inline core/kazoo_endpoint/src/kz_endpoint.erl]: Shouldn't this be a call to `is_true`?

Comment (sergey-safarov) [inline core/kazoo_endpoint/src/kz_endpoint.erl]: I will update after 1 hour

**Comment by lazedo** [comment]
@sergey-safarov we still don't see why we would change the defaults in freeswitch and this PR  depends on it. maybe close this in favour of #3872 (and fix the merge code) ?

**Pull request closed without merging**